pages/api/network/index.ts

```diff
@@ -459,8 +456,7 @@ async function put(req: NextApiRequest, res: NextApiResponse) {
         const { data } = await octokitUser.rest.repos.addCollaborator({
```

Comment:
we should not need this add collaborator logic anymore, because we are dropping github

Reply:
this logic was removed in [Pull Request](https://github.com/layerx-labs/webnetwork/pull/140)